contents/docs/deployment/deploy-docker.md

```diff
@@ -61,6 +61,28 @@ docker-compose up -d
 ```
 1. You're good to go! PostHog should be accessible on the domain you set up or the IP of your instance.
 
+### Configure docker-compose.yml
+
+You can also go through production-ready `docker-compose.yml` setup using our `docker-compose-config.py` configurator. It will guide you through relevant settings and explain them along the way. How to use it:
+
+1. Install PostHog's Python dependencies (preferably in a virtual environment).
+```bash
+python3 -m pip install -r requirements.txt
```

Comment:
Is this not a little problematic? If I'm running Docker, I want it to be a no-setup thing, but if I need to download all the dependencies that Docker will later download for me anyway (again, because it'll happen in the container), just so I can then run a CLI to generate my docker-compose file, I might as well just edit the file manually, no?

Reply:
I'm all for the generator, but I'm wondering now if we can consider a few things:
1. Tell the user to install just what they need for this script (I think from that script only `yaml` isn't in the standard library?)
2. Rewrite it as a shell script (Dealing with YAML is a bit annoying but we can just use a boilerplate and modify a few things)
3. Write a script that downloads the needed packages only and then runs the CLI 
4. Add a separate requirements file for this

I think 1 should be the way to go here (it's just the simplest) so could you maybe consider that?